readme.md

```diff
@@ -0,0 +1 @@
+# You need some documentation, asshole
```

Comment:
I commented it, are those notes not good enough?

Reply:
gotta have that markdown file bro

Reply:
aight there check that shit out now